web/app/js/components/BaseTable.jsx

```diff
@@ -0,0 +1,32 @@
+import { Table } from 'antd';
+
+export default class BaseTable extends Table {
```

Comment:
I think we should document that this class exists to override the toggle behaviour from Ant, since it's not super obvious that that's what we're doing. 
And maybe link to the overridden function in https://github.com/ant-design/ant-design/blob/master/components/table/Table.tsx#L348

Reply:
Good call, I left a little blob about it it above the component description 👍